pileface.py

```diff
@@ -1,16 +0,0 @@
-import random
-
-def pileFace():
```

Comment:
In python, we use snake case to define variable and function names : you can read more about frequent naming conventions [here](https://khalilstemmler.com/blogs/camel-case-snake-case-pascal-case/)